### ISSUE TITLE
No data found in the selected time range.

### ISSUE THREAD

**Issue by scottzockoll** [noitcudni/google-takeout-mileage-and-timeline-exporter-extension#8]
Hello i'm getting the following error: "No data found in the selected time range." when uploading my location-history.json file: 

<img width="743" alt="image" src="https://github.com/user-attachments/assets/903836c2-3c04-461f-9225-f8bcf59a2a92">

I can confirm that my location-history.json file has data in it from that time range.

**Comment by noitcudni** [comment]
Do you mind emailing me your json file. What’s the time range? I can look
into it. My email is lihster.developer@gmail.com.

**Comment by scottzockoll** [comment]
Yeah, I just sent it over with the specific location data redacted.
